source/adapters/hip/program.hpp

```diff
@@ -27,6 +27,12 @@ struct ur_program_handle_t_ {
   ur_device_handle_t Device;
   std::string ExecutableCache;
 
+  /* The ur_program_binary_type_t property is defined individually for every
```

Comment:
Note - this comment was copied from CUDA but appears out of date. I've left it as-is for now.